src/chrome/content/rules/nasdaqtrader.com.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="nasdaqtrader.com">
+	<target host="nasdaqtrader.com" />
+	<target host="www.nasdaqtrader.com" />
```

Comment:
See also:
```
beta.nasdaqtrader.com
classic.nasdaqtrader.com
ftp.nasdaqtrader.com
m.nasdaqtrader.com
tickpilotappendixb.nasdaqtrader.com
```

Reply:
Those have mismatch on cert. I'll add in header comment